subsys/net/l2/openthread/openthread.c

```diff
@@ -232,7 +232,6 @@ static enum net_verdict openthread_recv(struct net_if *iface,
 enum net_verdict openthread_send(struct net_if *iface, struct net_pkt *pkt)
 {
 	struct openthread_context *ot_context = net_if_l2_data(iface);
-	enum net_verdict ret = NET_OK;
```

Comment:
Change the commit title prefix to "net: openthread: "

Reply:
Done!